protocol/lib/context.go

```diff
@@ -11,6 +11,13 @@ import (
 	"github.com/dydxprotocol/v4-chain/protocol/lib/log"
 )
 
+// Custom exec modes
+const (
+	ExecModeBeginBlock        = 100
```

Comment:
using ints for these values?

Reply:
these are meant to augment [these](https://github.com/cosmos/cosmos-sdk/blob/6e9528a2f5e71b38e4866c57b4e9efd82ee4ee12/types/context.go#L18-L31)